### DISCUSSION TITLE
KERI Implementer's Call 06/20/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#89] [Meetings]
Reports
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Please add comments ahead of tomorrow's meeting for any additional topics

https://us06web.zoom.us/rec/share/rr7Mz33r9LKKkZiVuG0lHWKQROJgogqSN0eRMIzr7VFHYjB8DIY73CZLRndxlBND.k8vKhMQftVGDeZCg
Passcode: i39XcY6@

**Comment by pfeairheller** [comment]
### SigPy

- No new issues or PRs

**Comment by 2byrds** [comment]
KERIA
- Want to create a KERIA dev tag using the latest keripy dev tag and that allows all SIGNIFY-TS tests to pass (see next bullet) and enables a new dev tag for signify-ts
- Waiting on [PR review](https://github.com/WebOfTrust/keria/pull/254) that allows all [Signify-TS integration tests](https://github.com/WebOfTrust/signify-ts/pull/267) to pass
- Additional PR coming for [resubmitting for witness receipts](https://github.com/WebOfTrust/keria/issues/235)
- Discuss the recipient field in keripy (Sam will be adding to spec as well). As seen in Signify-TS and KERIA:
<img width="428" alt="image" src="https://github.com/WebOfTrust/keri/assets/681493/6ef16045-39ba-4e16-bdb9-f8f760a082a0">

**Comment by m00sey** [comment]
KERIpy:

- https://github.com/WebOfTrust/keripy/pull/798
- https://github.com/WebOfTrust/keripy/pull/805

**Comment by kentbull** [comment]
## SigTS
### Existing PRs
- 9 PRs, including single-sig IPEX apply, offer, and agree
### New PRs
- [Fix/test del rot multisig more](https://github.com/WebOfTrust/signify-ts/pull/267) #267


## CESRide
Nothing new the last week
